client/my-sites/plugins/notices.jsx

```diff
@@ -3,160 +3,153 @@
+		const logPlugin = this.getPluginById( sampleLog.pluginId );
 
-	refreshPluginNotices = () => {
-		const site = this.props.selectedSite;
 		return {
-			errors: filterNotices( PluginsLog.getErrors(), site, this.props.pluginSlug ),
-			inProgress: filterNotices( PluginsLog.getInProgress(), site, this.props.pluginSlug ),
-			completed: filterNotices( PluginsLog.getCompleted(), site, this.props.pluginSlug ),
+			plugin: logPlugin?.name,
+			numberOfPlugins,
+			site: logSite?.title,
+			isMultiSite: logSite?.is_multi_site,
+			numberOfSites,
 		};
-	};
+	}
 
 	showNotification = () => {
-		const logNotices = this.refreshPluginNotices();
-		this.setState( { notices: logNotices } );
-		if ( logNotices.inProgress.length > 0 ) {
-			notices.info(
-				this.getMessage( logNotices.inProgress, this.inProgressMessage, 'inProgress' )
+		const currentNotices = this.getCurrentNotices();
+
+		if ( currentNotices.inProgress.length > 0 ) {
+			this.props.infoNotice(
+				this.getMessage( currentNotices.inProgress, this.inProgressMessage, 'inProgress' ),
+				{
+					id: 'plugin-notice',
+				}
 			);
 			return;
 		}
 
-		if ( logNotices.completed.length > 0 && logNotices.errors.length > 0 ) {
-			notices.warning( this.erroredAndCompletedMessage( logNotices ), {
-				onRemoveCallback: () => reduxDispatch( removePluginStatuses( 'completed', 'error' ) ),
-			} );
-		} else if ( logNotices.errors.length > 0 ) {
-			notices.error( this.getMessage( logNotices.errors, this.errorMessage, 'error' ), {
-				button: this.getErrorButton( logNotices.errors ),
-				href: this.getErrorHref( logNotices.errors ),
-				onRemoveCallback: () => reduxDispatch( removePluginStatuses( 'error' ) ),
-			} );
-		} else if ( logNotices.completed.length > 0 ) {
-			const sampleLog =
-				logNotices.completed[ 0 ].status === 'inProgress'
-					? logNotices.completed[ 0 ]
-					: logNotices.completed[ logNotices.completed.length - 1 ];
-			// the dismiss button would overlap the link to the settings page when activating
-			const showDismiss =
-				sampleLog.action !== 'ACTIVATE_PLUGIN' ||
-				! get( sampleLog, 'plugin.wp_admin_settings_page_url' );
-
-			notices.success( this.getMessage( logNotices.completed, this.successMessage, 'completed' ), {
-				button: this.getSuccessButton( logNotices.completed ),
-				href: this.getSuccessHref( logNotices.completed ),
-				onRemoveCallback: () => reduxDispatch( removePluginStatuses( 'completed' ) ),
-				showDismiss,
+		if ( currentNotices.completed.length > 0 && currentNotices.errors.length > 0 ) {
+			this.props.warningNotice( this.erroredAndCompletedMessage( currentNotices ), {
+				onDismissClick: () => this.props.removePluginStatuses( 'completed', 'error' ),
+				id: 'plugin-notice',
 			} );
+		} else if ( currentNotices.errors.length > 0 ) {
+			this.props.errorNotice(
+				this.getMessage( currentNotices.errors, this.errorMessage, 'error' ),
+				{
+					onDismissClick: () => this.props.removePluginStatuses( 'error' ),
```

Comment:
We're intentionally using the redux action here directly, because this is now a connected component. No need to complicate things with `redux-bridge` anymore.